ios/FluentUI.Demo/FluentUI.Demo/Demos/DrawerVnextDemoController.swift

```diff
@@ -0,0 +1,181 @@
+
+        container.addArrangedSubview(UIView())
+
+        // Screen edge gestures to interactively present side drawers
+
+        let leadingEdgeGesture = UIScreenEdgePanGestureRecognizer(target: self, action: #selector(handleScreenEdgePan))
+        leadingEdgeGesture.edges = view.effectiveUserInterfaceLayoutDirection == .leftToRight ? .left : .right
+        view.addGestureRecognizer(leadingEdgeGesture)
+        navigationController?.navigationController?.interactivePopGestureRecognizer?.require(toFail: leadingEdgeGesture)
+
+        let trailingEdgeGesture = UIScreenEdgePanGestureRecognizer(target: self, action: #selector(handleScreenEdgePan))
+        trailingEdgeGesture.edges = view.effectiveUserInterfaceLayoutDirection == .leftToRight ? .right : .left
+        view.addGestureRecognizer(trailingEdgeGesture)
+    }
+
+    @objc private func showLeftDrawerClearBackgroundButtonTapped(sender: UIButton) {
+        delegate?.showLeftDrawerClearBackgroundSelected()
+    }
+
+    @objc private func showLeftDrawerDimmedBackgroundButtonTapped(sender: UIButton) {
+        delegate?.showLeftDrawerDimmedBackgroundSelected()
+    }
+
+    @objc private func showRightDrawerClearBackgroundButtonTapped(sender: UIButton) {
+        delegate?.showRightDrawerClearBackgroundSelected()
+    }
+
+    @objc private func showRightDrawerDimmedBackgroundButtonTapped(sender: UIButton) {
+        delegate?.showRightDrawerDimmedBackgroundSelected()
+    }
+
+    @objc private func handleScreenEdgePan(gesture: UIScreenEdgePanGestureRecognizer) {
+        guard gesture.state == .began else {
+            return
+        }
+
+        let leadingEdge: UIRectEdge = view.effectiveUserInterfaceLayoutDirection == .leftToRight ? .left : .right
+        if leadingEdge == .left {
+            delegate?.showLeftDrawerDimmedBackgroundSelected()
+        } else {
+            delegate?.showRightDrawerDimmedBackgroundSelected()
+        }
+    }
+}
+
+public protocol DrawerPresentationDelegate: AnyObject {
+    func showLeftDrawerClearBackgroundSelected()
+    func showLeftDrawerDimmedBackgroundSelected()
+    func showRightDrawerClearBackgroundSelected()
+    func showRightDrawerDimmedBackgroundSelected()
+}
+
+// MARK: VNTDrawerDemoController
+
+class DrawerVnextDemoController: DemoController, DrawerPresentationDelegate {
+
+    private var drawerController: VXTDrawerController?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let presentationController = DrawerPresentingController()
+        presentationController.delegate = self
+
+        let controller = VXTDrawerController(contentView: DrawerContentController().containerForActionViews(),
+                                             presentationController: presentationController)
+        container.addArrangedSubview(controller.view)
+        self.drawerController = controller
+
+        controller.view.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
isn't it weird that client app need to do this?

Reply:
I think the the init() of the wrapper class is the best place for it.

Reply:
This is mostly required to override the default behavior of `UIStackView`. 
Ideally I would expect the client to directly present or transition to `VXTDrawerController` without the need of `DemoController` which is how I think Teams should consume it.

Reply:
I updated this to use `MSFDrawerVnext` extension instead of using view controller API's